Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -580,6 +601,11 @@ - (NSApplicationPresentationOptions)window:(NSWindow*)window
            NSApplicationPresentationAutoHideMenuBar;
 }
 
+- (void)windowDidChangeScreen:(NSNotification *)notification
+{
+    _displayHelper->SetCurrentScreen(self.window.screen);
```

Comment:
This should be removed - I'll make sure it gets deleted. It's leftover from an attempt to make this support multiple displays.